### PR TITLE
Generic CloudWatch Metric Check and Beanstalk Env Check

### PR DESCRIPTION
**Pull request by ajmath**
Adds two new checks.  One to check a generic cloudwatch metric and another to check a specific metric on an elastic beanstalk ELB.

### CODE REVIEW

**Comment by analytically** [comment]
Please fix the offenses: https://travis-ci.org/sensu-plugins/sensu-plugins-aws/jobs/92969183

**Comment by ajmath** [comment]
Is there an automated way to apply these suggestions?

--
Andrew Matheny

**Comment by analytically** [comment]
No don't think so, you could try a code formatter and running rake again, and then manually fixing the leftovers.

**Comment by ajmath** [comment]
`bundle exec rubocop -a` will automatically fix rubocop errors.

File: bin/check-beanstalk-elb-metric.rb [hunk 1]
```diff
@@ -0,0 +1,123 @@
+#! /usr/bin/env ruby
+#
+# check-cloudwatch-alarm
```
Comment (eheydrick) [inline bin/check-beanstalk-elb-metric.rb]: should be check-beanstalk-elb-metric

File: bin/check-beanstalk-elb-metric.rb [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+         short:       '-s N',
+         long:        '--statistics NAME',
+         default:     'Average',
+         description: 'CloudWatch statistics method'
+
+  option :unit,
+         short:       '-u UNIT',
+         long:        '--unit UNIT',
+         description: 'CloudWatch metric unit'
+
```
Comment (eheydrick) [inline bin/check-beanstalk-elb-metric.rb]: can you be consistent about the spacing here. there's really no need to align the values.

File: bin/check-beanstalk-elb-metric.rb [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+  option :critical,
+         description: 'Trigger a critical when value is over VALUE',
+         short: '-c VALUE',
+         long: '--critical VALUE',
+         proc:        proc(&:to_f),
```
Comment (eheydrick) [inline bin/check-beanstalk-elb-metric.rb]: extra spaces.

File: bin/check-cloudwatch-metric.rb [hunk 4]
```diff
@@ -0,0 +1,117 @@
+#! /usr/bin/env ruby
+#
+# check-cloudwatch-alarm
```
Comment (eheydrick) [inline bin/check-cloudwatch-metric.rb]: should be check-cloudwatch-metric

File: bin/check-cloudwatch-metric.rb [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+#
+# DESCRIPTION:
+#   This plugin retrieves the value of a cloudwatch metric and triggers
+#   alarms based on the threshold's specified
```
Comment (eheydrick) [inline bin/check-cloudwatch-metric.rb]: thresholds

File: bin/check-cloudwatch-metric.rb [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+         short:       '-s N',
+         long:        '--statistics NAME',
+         default:     'Average',
+         description: 'CloudWatch statistics method'
+
+  option :unit,
+         short:       '-u UNIT',
+         long:        '--unit UNIT',
+         description: 'CloudWatch metric unit'
+
```
Comment (eheydrick) [inline bin/check-cloudwatch-metric.rb]: spacing consistency.

File: bin/check-cloudwatch-metric.rb [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+  option :critical,
+         description: 'Trigger a critical when value is over VALUE',
+         short: '-c VALUE',
+         long: '--critical VALUE',
+         proc:        proc(&:to_f),
```
Comment (eheydrick) [inline bin/check-cloudwatch-metric.rb]: spacing

**Comment by eheydrick** [comment]
Left a few more comments

**Comment by ajmath** [comment]
Can this be merged now?

**Comment by analytically** [comment]
Awesome, thanks!

**Pull request merged**